src/components/landing.jsx

```diff
@@ -0,0 +1,88 @@
+import { NavLink, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGear, faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { fetchCompany } from '../API/dataAPI';
+import style from '../styles/Company.module.css';
+
+function Details({ symbol }) {
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const { sector } = location.state || {};
+  const { company } = useSelector((state) => state.company);
+
+  useEffect(() => {
+    dispatch(fetchCompany(symbol));
+  }, [dispatch, symbol]);
+
+  if (!Object.keys(company).length) {
+    return (
+      <section className="animated">
+        <header>
+          <div>
+            <NavLink to={`/sectors/${encodeURI(company)}`}>
+              <FontAwesomeIcon icon={faChevronLeft} />
+            </NavLink>
+          </div>
+          <div>
+            <h1>
+              Finance Details
+            </h1>
+          </div>
+          <div><FontAwesomeIcon icon={faGear} /></div>
+        </header>
+      </section>
+    );
+  }
+  const {
+    companyWrapper, companyContainer, companyTitle, companyChanges, companyInfo, companyDescription,
+  } = style;
+  const {
+    image, changes, companyName, price, currency, industry, description,
+  } = company;
+  return (
+    <section className="animated">
+      <header>
+        <div>
+          <NavLink to={`/sectors/${sector}`}>
+            <FontAwesomeIcon icon={faChevronLeft} />
+          </NavLink>
+        </div>
+        <div>Stock Screaner</div>
```

Comment:
- [ ] Nice work implementing the `Back` button :clap: However when click on it on the landing page it doesn't link to the company page, but to an undefined page. See the video below to get more understanding of the issue :point_down: 
<video src="https://github.com/mohammadnewton/react-capstone/assets/91142432/1556b5ff-8b23-46cd-9178-966bcec688bb" alt="understand issue video"/>